### PR TITLE
HPCC-15623 ES,BR Translations

### PR DESCRIPTION
**Pull request by miguelvazq**
Add ES,BR translations

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
FYI @richardkchapman

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 1]
```diff
@@ -378,12 +408,15 @@
...
     RawTextPage: "Texto original (página atual)",
+    ReallyWantToRemove: "Realmente quer remover?",
     RecordCount: "Numero de Registros",
     RecordLength: "Tamanho do Registro",
     Records: "Registros",
     RecordSize: "Tamanho do Registro",
     RecordStructurePresent: "Estrutura de registro disponível",
     Recover: "Recuperar",
+    RecoverTooltip: "Reiniciar workunit en pausa / estancado",
+    RedBook: "Livro Vermelho",
```
Comment (GordonSmith) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Should this be "Red Book" like the other translations?

**Comment by GordonSmith** [comment]
Looks fine - one question re Red Book.

**Comment by miguelvazq** [comment]
In Spanish the term "Red Book" does not translate the same. It would physically mean a book that is ref

File: esp/src/eclwatch/nls/pt-br/hpcc.js (hunk 1 above)
Comment (miguelvazq) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: I'll refer to David on this one

Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Yes, that is true. Red book is best left "Red Book".

**Comment by miguelvazq** [comment]
@GordonSmith @dehilsterlexis Modified "Red Book" as per David's comment.

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 2]
```diff
@@ -539,11 +590,14 @@
     title_TpThorStatus: "Estado de Thor",
     title_UserPermissions: "Permissões de usuário",
     title_UserQuery: "Permissões",
+    title_WorkunitScopeDefaultPermissions: "Permissão Padrão de Unidades de Trabalho",
     title_WUDetails: "Detalhes da Tarefa ECL",
     title_WUQuery: "Tarefas de ECL",
     To: "Para",
     ToDate: "Data Final",
     Toenablegraphviews: "Para ativar visão gráfica, por favor instalar o plugin",
+    TooManyFiles: "Demasiados archivos",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Actually, I got this one wrong. Should be "Inumeros Arquivos"

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 3]
```diff
@@ -153,19 +161,23 @@
     FetchingData: "Buscando dados...",
     fetchingresults: "Buscando Resultados",
     File: "Arquivo",
+    FileCluster: "Archivo Cluster",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: 8 more mistakes: archivo = arquivo. I speak english and portuguese every day and my brain gets them mixed up. There are 8 Archivos => Arquivos

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 4]
```diff
@@ -378,12 +408,15 @@
...
     QuotedTerminator: "Terminador em Aspas",
     RawTextPage: "Texto original (página atual)",
+    ReallyWantToRemove: "Realmente quer remover?",
     RecordCount: "Numero de Registros",
     RecordLength: "Tamanho do Registro",
     Records: "Registros",
     RecordSize: "Tamanho do Registro",
     RecordStructurePresent: "Estrutura de registro disponível",
     Recover: "Recuperar",
+    RecoverTooltip: "Reiniciar workunit en pausa / estancado",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Reniciar unidade de trabalho pausado / bloqueado

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 5]
```diff
@@ -398,7 +431,10 @@
     RenderedSVG: "Rendered SVG",
     RenderSVG: "Renderização SVG",
     Replicate: "Replicar",
+    ReplicateOffset: "Replicar o Deslocamento",
+    RepresentsASubset: "Representar un subconjunto de todas los casamientos. Usando un filtro correcto puede reducir el numero de casamientos",
     RequestSchema: "Schema da Busca",
+    RequiredForXML: "Obligatorio para spray data en XML",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Manditório para spray de dados em XML

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 6]
```diff
@@ -411,6 +447,7 @@
     Restore: "Restaurar",
     Resubmit: "Resubmeter",
     Resubmitted: "Submeter de novo",
+    ResubmitTooltip: "Someter workunit de nuevo ",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Submeter unidade de trabalho de novo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 7]
```diff
@@ -440,6 +477,8 @@
     Skew: "Distorção",
     SLA: "SLA",
     Slaves: "Slaves",
+    SlaveLogs: "Slave logs",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Logs escravos

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 8]
```diff
@@ -440,6 +477,8 @@
     Skew: "Distorção",
     SLA: "SLA",
     Slaves: "Slaves",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Escravos

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 9]
```diff
@@ -449,8 +488,10 @@
     somefile: "*::somefile",
     Source: "Fonte",
     SourceCode: "Código Fonte",
+    SourceLogicalFile: "Nombre logico",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Nome Lógica do arquivo Fonte

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 10]
```diff
@@ -485,13 +526,16 @@
...
+    TargetRowTagRequired: "Es necesario suministrar el código de fila",
     TargetScope: "Escopo de Destino",
     TargetWuid: "Unidade de trabalho alvo",
     Terminators: "Terminadores",
     TestPages: "Páginas de Teste",
     Text: "Texto",
+    TheReturnedResults: "Los resultados regresados ",
     ThorMasterAddress: "Endereço de metre thor",
     ThorNetworkAddress: "Endereço de Rede do Thor",
+    ThorProcess: "Proceso de Thor",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Processo de Thor

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 11]
```diff
@@ -485,13 +526,16 @@
     TargetClusters: "Aglomeragos de Destino(Alvos)",
     TargetName: "Nome do Destino",
     TargetNamePlaceholder: "algum::lógico::arquivo",
+    TargetRowTagRequired: "Es necesario suministrar el código de fila",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Tag Obrigatório de Registro de Destino

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 12]
```diff
@@ -485,13 +526,16 @@
...
     TargetScope: "Escopo de Destino",
     TargetWuid: "Unidade de trabalho alvo",
     Terminators: "Terminadores",
     TestPages: "Páginas de Teste",
     Text: "Texto",
+    TheReturnedResults: "Los resultados regresados ",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Os Resultados Devolvidos

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 13]
```diff
@@ -12,6 +13,7 @@ define(
     ActiveWorkunit: "Unidad activa",
     Activities: "Actividades",
     Activity: "Actividad",
+    ActivityMap: "Mapa de Actividades",
     ActualSize: "Tamaño real",
     Add: "Agregar",
     AddFile: "Añadir Archivo",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/es/hpcc.js]: Adicionar Arquivo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 14]
```diff
@@ -43,10 +45,14 @@
     BlobPrefix: "Prefixo do BLOB",
     Bottom: "fundo",
     Cancel: "Cancelar",
+    CheckAllNodes: "Verifique Todos os Nós",
+    CheckFilePermissions: "Verifique Permissão de Archivo",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Verifique Permissão de Arquivo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 15]
```diff
@@ -153,19 +161,23 @@
     FetchingData: "Buscando dados...",
     fetchingresults: "Buscando Resultados",
     File: "Arquivo",
+    FileCluster: "Archivo Cluster",
     FileCounts: "Número de Arquivos",
     FileName: "Nome do Arquivo",
     FileParts: "Partes do Arquivo",
     FilePath: "Caminho e nome do Arquivo",
+    FilePermission: "Permissão de Archivo",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Permissão de Arquivo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 16]
```diff
@@ -153,19 +161,23 @@
...
     Files: "Arquivos",
     FileScopes: "Escopo de arquivos",
+    FileScopeDefaultPermissions: "Permissão Padrão de Archivo Escopo",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Permissão Padrão de Arquivo Escopo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 17]
```diff
@@ -306,11 +329,14 @@
...
     OpenInNewPageNoFrame: "Abrir em página nova (sem quadro)",
     OpenLegacyECLWatch: "Abrir Antigo ECL Watch",
+    OpenLegacyMode: "Modo Legado Aberto",
+    OpenNativeMode: "Modo Nativo Aberto",
     OpenSafeMode: "Abrir em Modo de Segurança",
     OpenSource: "Código Público",
     OpenTreeMode: "Abrir (modo árvore)",
     Operations: "Operações",
     Options: "Opções",
+    OriginalFile: "Archivo Originál",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Arquivo Originál

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 18]
```diff
@@ -338,12 +364,16 @@
     PctComplete: "% Completo",
     PercentDone: "Percentual Completo",
     PerformingLayout: "Executando Layout...",
+    Permission: "Permissão",
     PermissionName: "Nome da Permissão",
     Permissions: "Permissões",
+    PhysicalFiles: "Archivos Físicos",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Arquivos Físicos

**Comment by miguelvazq** [comment]
Changes made.

**Comment by richardkchapman** [comment]
@dehilsterlexis You ok with this now?

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 19]
```diff
@@ -251,12 +270,15 @@
...
     Low: "Baixo",
+    ManagedBy: "Gerenciado Por",
+    ManagedByPlaceholder: "CN=HPCCAdmin,OU=users,OU=hpcc,DC=MyCo,DC=local",
     ManualCopy: "Pressione Ctrl+C",
     ManualOverviewSelection: "(Seleção manual será necessária)",
     ManualTreeSelection: "(Seleção manual de árvore será requerida)",
     Mappings: "Mapeamento",
     Mask: "Mask (bloqueador de caracteres)",
     Max: "Max",
+    MaximumNumberOfSlaves: "Numero de esclavo",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Número de esclavo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 20]
```diff
@@ -439,7 +476,9 @@
     Size: "Tamanho",
     Skew: "Distorção",
     SLA: "SLA",
-    Slaves: "Slaves",
+    Slaves: "Escravos",
+    SlaveLogs: "Logs escravos",
+    SlaveNumber: "Numero de esclavo",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Número de esclavo

File: esp/src/eclwatch/nls/pt-br/hpcc.js [hunk 21]
```diff
@@ -539,11 +590,14 @@
     title_TpThorStatus: "Estado de Thor",
     title_UserPermissions: "Permissões de usuário",
     title_UserQuery: "Permissões",
+    title_WorkunitScopeDefaultPermissions: "Permissão Padrão de Unidades de Trabalho",
     title_WUDetails: "Detalhes da Tarefa ECL",
     title_WUQuery: "Tarefas de ECL",
     To: "Para",
     ToDate: "Data Final",
     Toenablegraphviews: "Para ativar visão gráfica, por favor instalar o plugin",
+    TooManyFiles: "Inumeros Arquivos",
```
Comment (dehilsterlexis) [inline esp/src/eclwatch/nls/pt-br/hpcc.js]: Inúmeros Arquivos

**Comment by dehilsterlexis** [comment]
@miguelvazq back to you. A few accents.

**Pull request merged**